apps/openchallenges/challenge-service/src/main/resources/db/challenges.csv

```diff
@@ -2,7 +2,7 @@
-"4","drug-sensitivity-and-drug-synergy-prediction","Drug Sensitivity and Drug Synergy Prediction","Predicting drug sensitivity in human cell lines","Development of new cancer therapeutics currently requires a long and protracted process of experimentation and testing. Human cancer cell lines represent a good model to help identify associations between molecular subtypes, pathways, and drug response. In recent years there have been several efforts to generate genomic profiles of collections of cell lines and to determine their response to panels of candidate therapeutic compounds. These data provide the basis for the development of in silico models of sensitivity based either on the unperturbed genetic potential of a cancer cell, or by using perturbation data to incorporate knowledge of actual cell response. Making predictions from either of these data profiles will be beneficial in identifying single and combinatorial chemotherapeutic response in patients. To that end, the present challenge seeks computational methods, derived from the molecular profiling of cell lines both in a static state and in response to perturbation of ...","","https://www.synapse.org/#!Synapse:syn2785778","completed","1","","2012-06-01","2012-10-01","2813","2023-11-01 22:08:36","2024-03-04 18:31:14"
+"4","drug-sensitivity-and-drug-synergy-prediction","Drug Sensitivity and Drug Synergy Prediction","Predicting drug sensitivity in human cell lines","Development of new cancer therapeutics currently requires a long and protracted process of experimentation and testing. Human cancer cell lines represent a good model to help identify associations between molecular subtypes, pathways, and drug response. In recent years there have been several efforts to generate genomic profiles of collections of cell lines and to determine their response to panels of candidate therapeutic compounds. These data provide the basis for the development of in silico models of sensitivity based either on the unperturbed genetic potential of a cancer cell, or by using perturbation data to incorporate knowledge of actual cell response. Making predictions from either of these data profiles will be beneficial in identifying single and combinatorial chemotherapeutic response in patients. To that end, the present challenge seeks computational methods, derived from the molecular profiling of cell lines both in a static state and in response to perturbation of ...","","https://www.synapse.org/#!Synapse:syn2785778","completed","1","","2012-06-01","2012-10-01","#N/A","2023-11-01 22:08:36","2024-03-04 18:31:14"
```

Comment:
Looks like the EDAM operation mapping broke in the OC Data Sheet, e.g. [cell Z5](https://docs.google.com/spreadsheets/d/1XRSUVqbZk_k9_rV1vpgPVMQ0fVV0mavwY8Aqyy0C2HQ/edit#gid=1834177142). @gaiaandreoletti - can you let me know what changes were done recently?

Reply:
4/3 update:

Reverted some changes made to the OC Data Sheet, which should fix this issue.